### ISSUE TITLE
ts-rainbow throws an error when loading

### ISSUE THREAD

**Issue by folke** [p00f/nvim-ts-rainbow#22]
I get the error below when opening a typescript file. Using the latest version of nvim-ts-rainbow.

```
Error detected while processing FileType Autocommands for "*":
E5108: Error executing lua ...ck/packer/start/nvim-ts-rainbow/lua/rainbow/internal.lua:45: attempt to index local 'query' (a nil value)
```

**Comment by folke** [comment]
It seems like https://github.com/p00f/nvim-ts-rainbow/pull/21 will fix the error

**Comment by p00f** [comment]
Closed by #21
